js/package.json

```diff
@@ -21,7 +21,7 @@
     "Parity"
   ],
   "scripts": {
-    "build": "npm run build:lib && npm run build:app",
+    "build": "npm run build:lib && npm run build:app && npm run build:embed",
```

Comment:
The only issue here is line 27 - fails on Windows.

Reply:
Solved by using `cross-env`